internal/analytics/client.go

```diff
@@ -83,27 +91,71 @@ func (c *Client) SendEvent(
 	}()
 }
 
+func (c *Client) SendCommandRunEvent(
+	accessToken,
+	baseURI string,
+	optOut bool,
+	properties map[string]interface{},
+) {
+	c.sendEvent(
+		accessToken,
+		baseURI,
+		optOut,
+		"CLI Command Run",
+		properties,
+	)
+	if !optOut {
```

Comment:
I'm not sure this adds anything since `SendCommandCompletedEvent` will call `c.sendEvent` which will no-op if optOut is true.

Reply:
True, I'll address this in a follow up